Cabal/src/Distribution/Simple/GHCJS.hs

```diff
@@ -1436,13 +1435,17 @@ gbuild verbosity numJobs pkg_descr lbi bm clbi = do
     sequence_
       [ do
         let baseCxxOpts =
-              Internal.componentCxxGhcOptions
-                verbosity
-                lbi
-                bnfo
-                clbi
-                tmpDir
-                filename
+              ( Internal.sourcesGhcOptions verbosity lbi bnfo clbi tmpDir filename
+                  <> mempty
+                    { ghcOptCcOptions =
```

Comment:
```haskell
              ( Internal.sourcesGhcOptions verbosity lbi bnfo clbi tmpDir filename
                  <> mempty
                    { ghcOptCcOptions =
```
is it not the same as
```suggestion
              ( (Internal.sourcesGhcOptions verbosity lbi bnfo clbi tmpDir filename)
                  { ghcOptCcOptions =
```
? (I hope I'm not making up this syntax!)

Reply:
Yes, you are right, it should be done the same for GHC and GHCJS, it should be done without `mempty`, because it is a replacement, not an addition.